eng/ci/templates/official/jobs/run-benchmarks.yml

```diff
@@ -16,10 +16,12 @@ jobs:
   variables:
     runDescription: ${{ parameters.description }}
     functionApp: ${{ parameters.functionAppName }}
-    functionAppOutputPath: $(Build.ArtifactStagingDirectory)\FunctionApps\$(functionApp)
-    benchmarkResultsJsonPath: "$(Build.ArtifactStagingDirectory)\\BenchmarkResults\\$(functionApp).json"
+    functionAppOutputPath: $(Build.ArtifactStagingDirectory)/FunctionApps/$(functionApp)
+    benchmarkResultsJsonPath: "$(Build.ArtifactStagingDirectory)/BenchmarkResults/$(buildNumber)_$(functionApp).json"
     functionsWorkerRuntime: 'dotnet-isolated'
     crankAgentUrl: "http://localhost:5010"  # Default crank agent URL.
+    configFilePath: "./eng/perf/http.benchmarks.yml"
+    hostLocation: "./../../"
```

Comment:
Can you make this an absolute path instead? Using something like `$(Build.SourcesDirectory)`

Reply:
No, when using a local folder, you cannot give an absolute path. Perhaps a future improvement crank should make. I will file an issue on that repo.